src/audio/player.js

```diff
@@ -316,1 +1,316 @@
+const {
+    AudioPlayerStatus,
+    createAudioPlayer,
+    createAudioResource,
+    entersState,
+    joinVoiceChannel,
+    VoiceConnectionStatus,
+} = require('@discordjs/voice');
+const Discord = require('discord.js');
+const play = require('play-dl');
+const log = require('../logger.js');
+const colors = require('../color.json');
+
+class Player {
+    /**
+   * @param {Discord.Message} event
+   * @param {Discord.Guild} guild
+   * @param {Discord.VoiceChannel} voice
+   */
+    constructor(event, guild, voice) {
+        this._client = event.client;
+        this._channel = event.channel;
+        this._guild = guild;
+        this._guildId = guild.id;
+        this._voiceChannel = voice;
+        this._channelId = voice.id;
+
+        this._init = false;
+        this._noticeMessage = null;
+        this._nowplaying = null;
+        this._songs = [];
+    }
+
+    static getSendingPlayer(guild) {

```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/blackcatbot/blackcat/pullRequest?prid=8779465)
```suggestion
    static getSendingPlayer(guild) {
```